### ISSUE TITLE
Is VS 2019 supported?

### ISSUE THREAD

**Issue by bhardwajs** [Azure/azure-storage-cpp#274]
Documentation says

> Via Vcpkg You can manage the dependencies with Vcpkg, and use Visual Studio 2015 update 3 or Visual Studio 2017 for development environment. Simply install Casablanca via Vcpkg will setup everything needed.

What about VS 2019? Can I use vcpkg with VS 2019?

**Comment by katmsft** [comment]
Yes, vcpkg automatically supports VS2019. However, although unlikely, there might be some build issues for this repository to resolve since we have not yet officially supported VS2019.

**Comment by bhardwajs** [comment]
I was able to build and test using VS2019.

How about connecting on MSFT internal to see what needs to be done to make VS 2019 officially supported?

**Comment by katmsft** [comment]
Normally we only supports two VS versions, we would need to discuss internally if we can drop VS 2015 and supports only VS 2017/2019. Please stay tuned.

**Comment by mloskot** [comment]
@katmsft  
> we would need to discuss internally if we can drop VS 2015 and supports only VS 2017/2019.

Any news on this?

**Comment by Frafjord** [comment]
In 16.10 it will give a warning about the use of std::atomic_load_explicit (deprecated in C++ 20).
